internal/pkg/manifest/rd_web_svc.go

```diff
@@ -60,15 +61,48 @@ func (c *RequestDrivenWebServiceNetworkConfig) IsEmpty() bool {
 	return c.VPC.isEmpty()
 }
 
-// RequestDrivenWebServicePlacement represents where to place tasks for a Request-Driven Web Service.
-type RequestDrivenWebServicePlacement Placement
+// RequestDrivenWebServicePlacement represents what types of subnets to place tasks.
+type RequestDrivenWebServicePlacementString PlacementString
+
+// RequestDrivenWebServicePlacementArgOrString represents where to place tasks for a Request-Driven Web Service.
+type RequestDrivenWebServicePlacementArgOrString struct {
+	PlacementString *RequestDrivenWebServicePlacementString
+	PlacementArgs
```

Comment:
Does this mean you will allow folks to specify the subnet IDs for the VpcConnector resource as well?

Reply:
Yeah that's right.